doc/man3/OSSL_CMP_CTX_new.pod

```diff
@@ -780,6 +788,8 @@ OSSL_CMP_CTX_set0_trustedStore() was renamed to OSSL_CMP_CTX_set0_trusted(),
 using macros, while keeping the old names for backward compatibility,
 in OpenSSL 3.1.
 
+OSSL_CMP_CTX_reset_geninfo_ITAVs() was added in OpenSSL 3.0.6.
```

Comment:
Not sure about the version 3.0.6 listed here is really right for master..

Reply:
Why not? IMO it is right. Unless we were in situation where master would be released earlier than the 3.0.6 version in which case it would be debatable. But that is definitely not the case.

Reply:
I also see this as correct.
In contrast, it certainly would be wrong to state "was added in OpenSSL 3.1"  because in fact it is added already for 3.0.6.